src/coreclr/nativeaot/Common/src/System/Collections/Generic/LowLevelList.cs

```diff
@@ -32,16 +34,14 @@ namespace System.Collections.Generic
     // Data size is smaller because there will be minimal virtual function table.
     // Code size is smaller because only functions called will be in the binary.
     [DebuggerDisplay("Count = {Count}")]
-#if TYPE_LOADER_IMPLEMENTATION
     [System.Runtime.CompilerServices.ForceDictionaryLookups]
-#endif
     internal class LowLevelList<T>
```

Comment:
Should the type be moved under TypeLoader since it's only required for it?

Additional question: is `ForceDictionaryLookups` only applicable for reference types because of generic sharing, or also applicable for value types?

Reply:
> Should the type be moved under TypeLoader since it's only required for it?

It is very likely that there is code in CoreLib that is used by the TypeLoader indirectly that would need to be annotated with this attribute if we ever resurrected this (https://github.com/dotnet/runtime/issues/85184#issuecomment-1545059247).

I think it would be ok to delete ForceDictionaryLookups and its few remaining uses. It is likely heavily bitrotten.

Reply:
Yeah, in this PR I was using `Array.Resize` which is generic. It seems that the real foreseeing solution is to limit TypeLoader usage of CoreLib, according to the linked discussion. It also means that TypeLoader needs its specialized collection. If we resurrected the generic lookup work, is it expected to use the attribute, or just do some specialized handling for TypeLoader?

The original question is that, would there be any place else (not involved with TypeLoader) unable to use the CoreLib collection? I assume no.

Reply:
Per https://github.com/dotnet/runtime/issues/85184#issuecomment-1545059247, if we ever resurrected the lazy generic lookup work, we would need likely ended up with TypeLoader in its own partition that does not perform the lazy generic lookups. It would be very large refactoring.